GVFS/GVFS.Service/GVFSMountProcess.cs

```diff
@@ -58,7 +58,10 @@ public void Dispose()
 
         private bool CallGVFSMount(string repoRoot)
         {
-            return this.CurrentUser.RunAs(Configuration.Instance.GVFSLocation, "mount " + repoRoot);
+            return this.CurrentUser.RunAs(
+                Configuration.Instance.GVFSLocation,

```

Comment:
This expression is now quite long with lots of string appending. Makes it hard to see if the spaces are in the correct place to separate arguments/values.

Consider making this an `$"interpolated {string}"` (or use a `StringBuilder`) and pull the final formatted string out of the `RunAs` call.